coherence/doc.go

```diff
@@ -534,6 +535,115 @@ Lastly, when you create a Customer object you must set the Class value matching
 	    log.Fatal(err)
 	}
 
+# Using Near Caches
+
+The Coherence Go client allows you to specify a near cache to cache frequently used and accessed data in your Go application.
+When you access data using Get() or GetAll() operations, returned entries are stored in the near cache and subsequent data
+access for keys in the near-cache is almost instantaneous.
+
+On creating a near cache, Coherence automatically adds a [MapListener] to your [NamedMap] or [NamedCache] which listens on
+all cache events and updates or invalidates entries in the near cache that have been changed or removed on the server.
+
+To manage the amount of memory used by the near cache, the following options are supported when creating one:
+
+  - time-to-live (TTL) – objects expired after time in near cache, e.g. 5 minutes
+  - High-Units – maximum number of cache entries in the near cache
+  - Memory – maximum amount of memory used by cache entries
+
+Note: You can specify TTL and High-Units or memory.
```

Comment:
Would it be clearer as "You can specify either High-Units or Memory and in either case, optionally, a TTL"?